src/core/localization/TranslationService.ts

```diff
@@ -7,6 +7,12 @@ import en_common from './translations/en/common.json';
 import es_common from './translations/es/common.json';
 // @ts-ignore
 import ar_common from './translations/ar/common.json';
+// @ts-ignore
+import ko_common from './translations/ko/common.json';
+// @ts-ignore
+import id_common from './translations/id/common.json';
```

Comment:
is 'id' a universal code for Indonesian?
i thought it was just an id first, then i thought it's Indian (then i remembered it's called hindi), and then i look at the PR description and found the right answer 😆

Reply:
yep, you're right =)